examples/hello/publisher/main.go

```diff
@@ -15,19 +15,38 @@ import (
 	"google.golang.org/grpc"
 )
 
+var url = flag.String("url", ":8080", "the grpc url to the pubsub server")
 var topic = flag.String("topic", hello.DefaultTopicName, "the topic to publish to")
 
 func main() {
 	flag.Parse()
-	conn, err := grpc.Dial(":8080", grpc.WithInsecure())
+	conn, err := grpc.Dial(*url, grpc.WithInsecure())
 	if err != nil {
 		log.Fatalf("failed to dial to grpc server: %v", err)
 	}
+<<<<<<< HEAD
```

Comment:
Uh-oh, this file has conflicts still...

Reply:
you're too quick! I noticed this after pushing to Roman's remote... I guess I still have to learn how to use this new merge tool of mine. Should be all fixed now.